src/app/create/route.ts

```diff
@@ -0,0 +1,87 @@
+import * as Yup from "yup";
+import { db, getErrorMessage } from "../../lib";
+import { Msr, MsrPiiSec } from "../../types";
+import { Gender, MSRStatus, Race } from "@prisma/client";
+
+const payloadSchema = Yup.object({
+	zendesk_user_id: Yup.number().required(),
+	email: Yup.string().email().required(),
+	phone: Yup.string().required(),
+	first_name: Yup.string().required(),
+	last_name: Yup.string(),
+	city: Yup.string().required(),
+	state: Yup.string().required(),
```

Comment:
```suggestion
	state: Yup.string().required().max(2),
```

add q o state tem q so ter duas letras